OpenRA.Mods.Common/Lint/CheckUnknownTraitFields.cs

```diff
@@ -37,7 +37,14 @@ void CheckActors(IEnumerable<MiniYamlNode> actors, Action<string> emitError, Mod
 					// Removals can never define children
 					if (t.Key.StartsWith("-", StringComparison.Ordinal) && t.Value.Nodes.Any())
 					{
-						emitError("{0} has child nodes, which is not valid for removals.".F(t.Key));
+						emitError("{0} defines child nodes, which are not valid for removals.".F(t.Location, t.Key));
```

Comment:
1 variable in string, but 2 in `.F()`?. `t.Key` isn't used. It probably should tell the trait name, tho i'm not good at writing sentences for those stuff.

Reply:
Looks to me like it should be `emitError("{0}: {1} defines ...`